src/packaging/packages.targets

```diff
@@ -60,14 +60,37 @@
             <ILCompilerSdkFiles Include="bootstrappercpp" />
             <ILCompilerSdkFiles Include="System.Private.CoreLib.Native" Condition="'$(OsEnvironment)'!='Windows_NT'" />
 
+            <!-- ILCompiler.SDK Cpp Codegen support files -->
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/env/etmdummy.h" />
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/env/gcenv.base.h" />
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/env/gcenv.object.h" />
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/env/gcenv.sync.h" />
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/gc.h" />
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/gcdesc.h" />
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/sample/gcenv.h" />
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/gcimpl.h" />
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/gcpriv.h" />
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/gcrecord.h" />
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/gcscan.h" />
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/handletable.h" />
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/handletable.inl" />
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/handletablepriv.h" />
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/objecthandle.h" />
+            <ILCompilerSdkCppCodegenFiles Include="Native/Bootstrap/common.h" />
+            <ILCompilerSdkCppCodegenFiles Include="Native/Bootstrap/platform.h" />
+
             <ILCompilerSdkFilesManaged Include="System.Private.CoreLib" />
+
             <ILCompilerSdkBinPlace Include="@(ILCompilerSdkFiles)">
                 <Text><![CDATA[        <file src="../lib/$(LibPrefix)%(Identity).$(StaticLibExt)" target="runtimes/$(NuPkgRid)/native/sdk/$(LibPrefix)%(Identity).$(StaticLibExt)" /> ]]></Text>
             </ILCompilerSdkBinPlace>
             <ILCompilerSdkBinPlace Include="@(ILCompilerSdkFilesManaged)">
                 <Text><![CDATA[        <file src="../%(Identity).dll" target="runtimes/$(NuPkgRid)/native/sdk/%(Identity).dll" /> ]]></Text>
             </ILCompilerSdkBinPlace>
-
+            <ILCompilerSdkBinPlace Include="@(ILCompilerSdkCppCodegenFiles)">
```

Comment:
NIT: Putting things inside ilc-sdk might be better, because in the cli/bin/ we get a single folder for us.